app/controllers/api/v1/materials_controller.rb

```diff
@@ -1,6 +1,21 @@
 class API::V1::MaterialsController < API::APIController
   include Materials::DataHelpers
 
+  #
+  # Map of class types supported material types.
+  #
+  # Might also consider using "classify" and "constantize" here
+  # rather than a hard coded map of types. But that might require
+  # additional input validation or otherwise constrain how we define
+  # the http parameters.
+  #
+  @@supported_material_types = {
+    "external_activity" => ExternalActivity,
+    "interactive"       => Interactive,
+    "investigation"     => Investigation
+  }
+
+
```

Comment:
I think you can remove 'Investigation' from this list unless you know of a reason otherwise.